common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1549,7 +1550,7 @@ def test_get_html(self):
         module = CapaFactory.create()
         module.get_progress = Mock(wraps=module.get_progress)
         module.get_html()
-        module.get_progress.assert_called_once_with()
+        module.get_progress.assert_called_with()
```

Comment:
I assume this means the method is now being called multiple times. Is it only being called the number of times that we expect it to be?